### DISCUSSION TITLE
下载CDN资源到本地

### DISCUSSION THREAD

**Discussion by Half-Sugar-Code** [cdnjs/cdnjs#14146] [General]
我有一个问题，我应该怎么实现使用axios库将cdn的资源下载到本地呢？谢谢

**Comment by MattIPv4** [comment]
Hi there,

We can't provide support with individual libraries, we just host them all!

If you're looking to just download CDN assets though, you should be able to just make a GET request to the assets to fetch them. You can also make use of our API to see what assets are available for each library: https://cdnjs.com/api

**Reply by Half-Sugar-Code** [reply]
OK , Thank you.
